src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -197,35 +199,115 @@ def __call__(self, input_ids, token_type_ids, position_ids, attention_mask, dete
         hidden_states = inputs_embeds + jnp.broadcast_to(position_embeds, inputs_embeds.shape) + token_type_embeddings
 
         # Layer Norm
-        hidden_states = self.layer_norm(hidden_states)
+        hidden_states = self.LayerNorm(hidden_states)
         hidden_states = self.dropout(hidden_states, deterministic=deterministic)
         return hidden_states
 
 
-class FlaxBertAttention(nn.Module):
+class FlaxBertSelfAttention(nn.Module):
     config: BertConfig
     dtype: jnp.dtype = jnp.float32  # the dtype of the computation
 
     def setup(self):
-        self.self_attention = nn.attention.SelfAttention(
-            num_heads=self.config.num_attention_heads,
-            qkv_features=self.config.hidden_size,
+        assert self.config.hidden_size % self.config.num_attention_heads == 0, "Error"
```

Comment:
A real error message please! Also this is going to take more than 119 chars, so be reformatted and in this case I find the explicit
```
if self.config.hidden_size % self.config.num_attention_heads != 0:
    raise ValueError("Fill me please Patrick!")
```
more readable.